safe-global/web-core
# add: a/b tested promotion of hiding spam tokens

## What it solves
- Adds a onboarding Tooltip to the new "hide token" button which will only be shown until dismissed.
- the tooltip will only be shown to 50% of users to a/b test their efficiency.


## How to test it
1. Open the assets of a Safe
2. If the tooltip is not shown, delete the `SAFE_v2__AB_hide_token_promo` entry in your local storage.
  2.1. Dismiss it => it should not come up again until the local storage is cleared

## Analytics changes
None

## Screenshots
![Screenshot 2023-01-09 at 16 45 01](https://user-images.githubusercontent.com/2670790/211350145-aae0d443-35a3-4d20-af01-8cf8330ea51f.png)
